packages/@okta/vuepress-site/docs/reference/password-hook/index.md

```diff
@@ -72,11 +72,11 @@ The following command is supported for the Registration Inline Hook type:
 
 The `value` object is the parameter to pass to the command.
 
-For `com.okta.action.update` commands, `value` should be an object containing a `credential` property set to either `VERIFIED` or `UNVERIFIED`.
+For `com.okta.action.update` commands, `value` should be an object containing a `credential` property set to either `VERIFIED` or `UNVERIFIED`:
```

Comment:
...an object that contains a `credential`......

for translation

Reply:
okay, made the change